wlib/EclCommand.h

```diff
@@ -161,6 +161,7 @@ class CEclCommandMixin : public CMessageMap
         UPDATEUI(cui, ID_ECL_CLEARERRORS, m_ecl->SyntaxGetCount() > 0);
         UPDATEUI(cui, ID_ECL_GOTO, bHasContents);
         UPDATEUI(cui, ID_ECL_SEL_GO, bHasSelection);
+        UPDATEUI(cui, ID_GO_SUBMITNOARCHIV, bCanSubmit);
```

Comment:
Should this be:  `ID_ECL_NOARCHIV_GO`?

Reply:
I think I have the names backwards when compared to the current naming convention.